server/aws-lsp-codewhisperer/src/language-server/agenticChat/agenticChatController.ts

```diff
@@ -1317,7 +1369,7 @@ export class AgenticChatController implements ChatHandlers {
         } else if (toolUse?.name === 'fsRead') {
             await this.#features.lsp.window.showDocument({ uri: params.filePath })
         } else {
-            const absolutePath = params.fullPath ?? (await this.#resolveAbsolutePath(params.filePath))
+            const absolutePath = params.filePath ?? (await this.#resolveAbsolutePath(params.filePath))
```

Comment:
what is this fix for?

Reply:
This was giving an error to a dependency update. Removed this change now that I've clean installed and the error is gone.